pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -151,6 +158,8 @@ func checkUserPartOfLibvirtGroup() (bool, error) {
 	}
 	cmd := exec.Command(path, currentUser.Username)
 	buf := new(bytes.Buffer)
+	cmd.Env = crcos.ReplaceEnv(os.Environ(), "LC_ALL", "C")
+	cmd.Env = crcos.ReplaceEnv(cmd.Env, "LANG", "C")
```

Comment:
This way too much iterative. I think we want something like in `crcos` side and every `exec` should use it.

```
func RunWithOutPrivilege(cmdName string, cmdAndArgs ...string) (string, string, error) {
	cmd := exec.Command(cmdName, cmdAndArgs...)
	stdOut := new(bytes.Buffer)
	stdErr := new(bytes.Buffer)
	cmd.Env = ReplaceEnv(os.Environ(), "LC_ALL", "C")
	cmd.Env = ReplaceEnv(cmd.Env, "LANG", "C")
	err := cmd.Run()
	return stdOut.String(), stdErr.String(), err

}
```

Reply:
Yup.. this is should be put into a function, Will update, Thanks!